packages/create-sitecore-jss/src/templates/angular-xmcloud/src/app/routing/layout/layout.component.html

```diff
@@ -0,0 +1,37 @@
+<app-navigation></app-navigation>
+<div class="{{ mainClassPageEditing }}">
+  <ng-container *ngIf="state === LayoutState.Layout">
+    <app-scripts></app-scripts>
+    <header>
+      <div id="header">    
+        <sc-placeholder
+          name="headless-header"
+          [rendering]="route"
```

Comment:
Add the `loaded` event handler (should be wired up for each placeholder - and think we would then leave the function name as it was "onPlaceholderLoaded").
```suggestion
          [rendering]="route"
          (loaded)="onPlaceholderLoaded($event)"
```